app/src/main/res/drawable/splash_screen_background.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@android:color/white"/>
+    <item>
+        <bitmap
+            android:gravity="fill_vertical|fill_horizontal"
+            android:src="@drawable/splash_image" />
+    </item>
+</layer-list>
```

Comment:
nit: could you insert a new line here?

Reply:
@SanketDG I will update the PR

Reply:
@SanketDG I tried to update the PR, but I don't know why the changes are being shown over here